music_methods.rb

```diff
@@ -0,0 +1,28 @@
+require_relative './app'
+require_relative './musicalbum'
+
+def list_music
+  @music_list.each_with_index do |music, index|
+    puts "(#{index}) ID: #{music.id} On Spotify: \"#{music.on_spotify}\", Publish Date: #{music.publish_date} Archived: #{music.archived}" # rubocop:disable Layout/LineLength
+  end
```

Comment:
- [ ] It would be better if the false value on the `On Spotify` gets displayed on the console. Currently, only the true value gets displayed as show in the screenshot below
![Screenshot from 2022-08-03 15-45-37](https://user-images.githubusercontent.com/46388113/182656111-6ea636c3-d49a-4a08-8421-6897c71bcfd5.png)

Reply:
Thanks for your review. We revised the `musicalbum.rb` to  `@on_spotify = on_spotify == 'y'`.